### PR TITLE
Service Fabric 8.2 Second Refresh Release Notes

### PR DESCRIPTION
**Pull request by sukanyamsft**
Service Fabric 8.2 Second Refresh Release Notes

### CODE REVIEW

File: release_notes/Service_Fabric_ReleaseNotes_82CU2.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Microsoft Azure Service Fabric 8.2 Second Refresh Release Notes
+
+* [Current Breaking Changes and Updated Support](#Current-Breaking-Changes-And-Updated-Support)
+* [Service Fabric Common Bug Fixes](#Service-Fabric-Common-Bug-Fixes)
+* [Repositories and Download Links](#Repositories-And-Download-Links)
+
+
+## Current Breaking Changes and Updated Support
+* **Guest Executable and Container Applications**: Guest executable and container applications created or upgraded in Service Fabric clusters with runtime versions 7.1+ are incompatible with prior Service Fabric runtime versions (e.g. Service Fabric 7.0).<br/>
```
Comment (peterpogorski) [inline release_notes/Service_Fabric_ReleaseNotes_82CU2.md:27]: Are Service Fabric 7.0 and 7.1 still considered supported?

File: release_notes/Service_Fabric_ReleaseNotes_82CU2.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+* **Windows Server 2022**: Service Fabric runtime will officially provide support for Windows Server 2022 starting with Service Fabric 8.1 CU3 release and all future releases.
+
+
+## Service Fabric Common Bug Fixes
+
+ | Versions | IssueType | Description | Resolution | 
+ | - | - | - | - | 
+ | **Windows - 8.2.1486.9590<br>Ubuntu 18 - 8.2.1285.1** | **Bug** | Disable FabricClient Cache in HTTP Gateway | **Brief Description**: Http gateway cache inside the FabricGateway.exe process does not automatically flush stale endpoint data from cache. As such, clients may end up receiving stale endpoints. <br> **Impact**: Client code calling http gateway can receive stale endpoints. <br> **Fix**: Disable the client cache inside http gateway. This forces http gateway to always query for the most recent data. <br> **Workaround**: A possible workaround is manually restarting the FabricGateway.exe process that is returning stale data. |
```
Comment (peterpogorski) [inline release_notes/Service_Fabric_ReleaseNotes_82CU2.md:50]: * Standardize on the capitalization of http

Comment (sukanyamsft) [reply release_notes/Service_Fabric_ReleaseNotes_82CU2.md]: Done!

File: release_notes/Service_Fabric_ReleaseNotes_82CU2.md (hunk 1 above)
Comment (sukanyamsft) [reply release_notes/Service_Fabric_ReleaseNotes_82CU2.md:27]: Unfortunately, we still have partners running on it. So, yes.

**Review by peterpogorski — APPROVED** [review]

**Pull request merged**